css/style.css

```diff
@@ -3,3 +3,21 @@
   padding: 0;
   box-sizing: border-box;
 }
+
+.navigation {
+  display: flex;
+  flex-direction: row;
```

Comment:
`flex-direction: row` is the default value, when using `display: flex;` You don't need that. :-)